packages/grid/_modules_/grid/components/cell/GridCell.tsx

```diff
@@ -114,6 +108,12 @@ export const GridCell: React.FC<GridCellProps> = React.memo((props) => {
     maxHeight: height,
   };
 
+  React.useLayoutEffect(() => {
+    if (hasFocus && cellMode === 'view' && cellRef.current) {
+      cellRef.current!.focus();
+    }
+  });
```

Comment:
At each reader it will refocus the cell container? Sounds like it will create bugs. For instance, what if I have a focusable checkbox in my cell? Happy to delay the resolution to the next bug a developer will open as it should allow us to add a test case for this. Now if we double check this point now, it should actually be better.